version/version.go

```diff
@@ -6,7 +6,7 @@ import (
 
 var (
 	// Version is the main version number that is being run at the moment.
-	Version = "0.6.3"
+	Version = "1.2.0"
```

Comment:
Is there a reason to update this? 🤔

Reply:
I think we haven't been updating it in a while. Current version is `1.1.0`. Should probably automate it at some point.

Reply:
Ohh, I've mistakenly thought this is a Packer API version bump after looking at the `go.mod` changes 🤦